gbus/worker.go

```diff
@@ -151,7 +151,7 @@ func (worker *worker) extractBusMessage(delivery amqp.Delivery) (*BusMessage, er
 	var decErr error
 	bm.Payload, decErr = worker.serializer.Decode(delivery.Body, bm.PayloadFQN)
 	if decErr != nil {
-		worker.log().WithError(decErr).WithField("message", delivery).Error("failed to decode message. rejected as poison")
+		worker.log().WithError(decErr).WithField("fqn", bm.PayloadFQN).Error("failed to decode message. rejected as poison")
```

Comment:
we use message_name as the field name to log when logging this in other places so let's align on that

```suggestion
		worker.log().WithError(decErr).WithField("meessage_name", bm.PayloadFQN).Error("failed to decode message. rejected as poison")
```